### ISSUE TITLE
TypeError with read_parquet

### ISSUE THREAD

**Issue by syagev** [dask/distributed#2632]
`read_parquet` seems to fail with a TypeError during serialization of tasks when run in dask distributed.

Minimal example:
```python
import dask
from dask import dataframe as dd
import pandas as pd
dask.config.set(scheduler='processes')

df = dd.from_pandas(pd.DataFrame({'a':list('abc'),'B':[1,2,3]}), npartitions=1)
df.to_parquet('a.parquet')
dd.read_parquet('a.parquet').compute()
```

This will raise the following error:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-ec034c5b8321> in <module>()
----> 1 dd.read_parquet('a.parquet').compute()

/usr/local/lib/python3.6/dist-packages/dask/base.py in compute(self, **kwargs)
    154         dask.base.compute
    155         """
--> 156         (result,) = compute(self, traverse=False, **kwargs)
    157         return result
    158 

/usr/local/lib/python3.6/dist-packages/dask/base.py in compute(*args, **kwargs)
    396     keys = [x.__dask_keys__() for x in collections]
    397     postcomputes = [x.__dask_postcompute__() for x in collections]
--> 398     results = schedule(dsk, keys, **kwargs)
    399     return repack([f(r, *a) for r, (f, a) in zip(results, postcomputes)])
    400 

/usr/local/lib/python3.6/dist-packages/dask/multiprocessing.py in get(dsk, keys, num_workers, func_loads, func_dumps, optimize_graph, pool, **kwargs)
    190                            get_id=_process_get_id, dumps=dumps, loads=loads,
    191                            pack_exception=pack_exception,
--> 192                            raise_exception=reraise, **kwargs)
    193     finally:
    194         if cleanup:

/usr/local/lib/python3.6/dist-packages/dask/local.py in get_async(apply_async, num_workers, dsk, result, cache, get_id, rerun_exceptions_locally, pack_exception, raise_exception, callbacks, dumps, loads, **kwargs)
    447             # Seed initial tasks into the thread pool
    448             while state['ready'] and len(state['running']) < num_workers:
--> 449                 fire_task()
    450 
    451             # Main loop, wait on tasks to finish, insert new ones

/usr/local/lib/python3.6/dist-packages/dask/local.py in fire_task()
    441                 # Submit
    442                 apply_async(execute_task,
--> 443                             args=(key, dumps((dsk[key], data)),
    444                                   dumps, loads, get_id, pack_exception),
    445                             callback=queue.put)

/usr/local/lib/python3.6/dist-packages/dask/multiprocessing.py in _dumps(x)
     24 
     25 def _dumps(x):
---> 26     return cloudpickle.dumps(x, protocol=pickle.HIGHEST_PROTOCOL)
     27 
     28 

/usr/local/lib/python3.6/dist-packages/cloudpickle/cloudpickle.py in dumps(obj, protocol)
    929     try:
    930         cp = CloudPickler(file, protocol=protocol)
--> 931         cp.dump(obj)
    932         return file.getvalue()
    933     finally:

/usr/local/lib/python3.6/dist-packages/cloudpickle/cloudpickle.py in dump(self, obj)
    282         self.inject_addons()
    283         try:
--> 284             return Pickler.dump(self, obj)
    285         except RuntimeError as e:
    286             if 'recursion' in e.args[0]:

/usr/lib/python3.6/pickle.py in dump(self, obj)
    407         if self.proto >= 4:
    408             self.framer.start_framing()
--> 409         self.save(obj)
    410         self.write(STOP)
    411         self.framer.end_framing()

/usr/lib/python3.6/pickle.py in save(self, obj, save_persistent_id)
    474         f = self.dispatch.get(t)
    475         if f is not None:
--> 476             f(self, obj) # Call unbound method with explicit self
    477             return
    478 

/usr/lib/python3.6/pickle.py in save_tuple(self, obj)
    734         if n <= 3 and self.proto >= 2:
    735             for element in obj:
--> 736                 save(element)
    737             # Subtle.  Same as in the big comment below.
    738             if id(obj) in memo:

.... (omitted for brevity)

/usr/lib/python3.6/pickle.py in save_dict(self, obj)
    819 
    820         self.memoize(obj)
--> 821         self._batch_setitems(obj.items())
    822 
    823     dispatch[dict] = save_dict

/usr/lib/python3.6/pickle.py in _batch_setitems(self, items)
    845                 for k, v in tmp:
    846                     save(k)
--> 847                     save(v)
    848                 write(SETITEMS)
    849             elif n:

/usr/lib/python3.6/pickle.py in save(self, obj, save_persistent_id)
    494             reduce = getattr(obj, "__reduce_ex__", None)
    495             if reduce is not None:
--> 496                 rv = reduce(self.proto)
    497             else:
    498                 reduce = getattr(obj, "__reduce__", None)

/usr/local/lib/python3.6/dist-packages/pyarrow/_parquet.cpython-36m-x86_64-linux-gnu.so in pyarrow._parquet.FileMetaData.__reduce_cython__()

TypeError: no default __reduce__ due to non-trivial __cinit__
```

Version info:

dask - 1.12.0
pyarrow - 0.13.0

See it live on [colab](https://colab.research.google.com/drive/1gL0u4Bq0nrH1FlGstT78UU3EQ_gr5ynA#scrollTo=_n-6DQEMaYdT)

**Comment by martindurant** [comment]
This is a known compatibility problem with arrow 0.13 - please downgrade.

Duplicate of https://github.com/dask/distributed/issues/2597 , closing this one.
